### ISSUE TITLE
Review handling of names

### ISSUE THREAD

**Issue by timgdavies** [openownership/data-standard#50]
See https://www.w3.org/International/questions/qa-personal-names and https://github.com/open-contracting/standard/issues/637#issuecomment-349346543

**Comment by ScatteredInk** [comment]
Something that has emerged from examining beneficial ownership declaration forms is that:

1. Legal names are not always the names represented on the shareholder register;
2. If a non-legal name is used on a  shareholder register, a legal name may still be required on beneficial ownership register and a discrepancy may emerge.

So, with names, we are dealing with two dimensions of category:

 1. The relationship of a name to a unique person (`alias`, `former` etc).
2. The relationship of a name to the other entities that we are describing.

We are implicitly saying now that a `name` in a `personStatement` matches across entities linked by a `groupID`, i.e., that we would expect the same name to appear in the shareholder's register as in a declaration of control. In reality these are two different sources that could differ.

So, instead, we might expect a `personStatement` to help us match the unique individual declaring a beneficial ownership interest using name `A`, but who is known on the shareholders' register as `A*`

I would then expect `names` to be an array of `Name` objects, similar to `AlternativeName` now, with a `type` field to account for the relationship to other modelled entities.

**Comment by ScatteredInk** [comment]
Our name types now are:

```       
[
    "detailed",
     "translation",
     "former",
     "alias",
     "birth"
 ]
```

There is a bit of work to do matching these with other standards, particularly in related fields.

OECD CRS uses these name types:

• OECD201= SMFAliasOrOther (not used for CRS)
• OECD202= indiv
• OECD203= alias
• OECD204= nick
• OECD205= aka
• OECD207= legal
• OECD208= atbirth

**Comment by timgdavies** [comment]
> I would then expect names to be an array of Name objects, similar to AlternativeName now, with a type field to account for the relationship to other modelled entities.

I had envisaged `name` as a field to use in cases where a source dataset *only* has a single name, and then `alternateNames` as a place for both (a) a breakdown of that name where available; and (b) a place to provide alternative names. 

However, I realise this may be conflating the goal of providing a field for simple systems, with recognising individuals often have name(s) rather than a name. 

That leads me to agree that we should just have `names` as an array of `AlternateName` objects (which we would rename to `Name`), and then guidance that simple cases can just use:

`names/0/fullName` 

but others should (a) provide the breakdown, and (b) provide other names where available. 

We would then either want to (a) state that the `names` array is ordered, and the preferred name for interfaces should be the first item in the array; or (b) add a flag or business logic rule about which name should be used as the display name.

**Comment by timgdavies** [comment]
There is another issue for us to explore also around historic names, particularly in the context of transgender communities, where exposure of past names can be a source of harm. C.f: https://www.pinknews.co.uk/2017/11/21/government-urged-to-change-law-that-outs-transgender-business-people/

**Comment by ScatteredInk** [comment]
Can we add `matronymicName` to match `patronymicName`?

This would cover, e.g., the [Chilean](https://en.wikipedia.org/wiki/Naming_customs_of_Hispanic_America) usage:

> Instead of primer apellido (first surname) and segundo apellido (second surname), common in other countries, legally, the following expressions are used: apellido paterno (paternal surname) and apellido materno (maternal surname). Both surnames are equally important and having two surnames is obligation for any person in birth registrations, the use of them are mandatory for any official document.

**Comment by timgdavies** [comment]
We can consider this, but I'm cautious about this step in the current updates - as it would take us away from the [European Union Core Persons Vocabulary](https://www.w3.org/ns/person#) - so we should review how they handled Chillean names. 

If Chilean usage specifies an ordering of these names, then both could be included in 'familyName'?

**Comment by ScatteredInk** [comment]
Good point. From reviewing EU Core, maternal and paternal family names are put into `familyName` and `patronymicName` remains unused.

For Chile, where order is required, we can just use `familyName` - but we might want to keep our eyes open for edge cases here. In other countries, order of names is not guaranteed, so it might be worth raising this with EU Core Persons issue tracker.

**Comment by timgdavies** [comment]
I've updated the issue title to reflect that the action here is most likely to be to improve our documentation of how to capture and record names. 

This may result in a new guidance page as part of an updated standard release.
